appinventor/components/src/com/google/appinventor/components/runtime/Sprite.java

```diff
@@ -181,8 +187,6 @@ public void Enabled(boolean enabled) {
    *
    * @param userHeading degrees above the positive x-axis
    */
-  @SimpleProperty(
```

Comment:
Why was this annotation removed? It results in the `set Heading` block disappearing from the blocks language.

Reply:
Total mistake. Thanks for catching it.

(In case your question wasn't rhetorical, it was a casualty of my moving Heading into the subclasses then improperly moving them back to the abstract superclass.)

Reply:
Mistake. Thanks for catching.